sample/src/main/AndroidManifest.xml

```diff
@@ -27,7 +27,8 @@
     <application
         android:label="@string/app_name"
         android:theme="@style/Theme.Catalog"
-        tools:ignore="MissingApplicationIcon">
+        android:icon="@mipmap/ic_launcher"
+        android:roundIcon="@mipmap/ic_launcher_round">
```

Comment:
`ic_launcher_round` content is identical to `ic_launcher`. We should either reuse the same file, or provide a different content.

Reply:
I'll provide a different one 👍